src/network/assemble.py

```diff
@@ -0,0 +1,33 @@
+"""
+Create bitmessage protocol command packets
+"""
+
+import struct
+
+import addresses
+from network.constants import MAX_ADDR_COUNT
+from network.node import Peer
+from protocol import CreatePacket, encodeHost
+
+
+def assemble_addr(peerList):
+    """Create address command"""
+    if isinstance(peerList, Peer):
+        peerList = (peerList)
+    if not peerList:
+        return b''
+    retval = b''
+    for i in range(0, len(peerList), MAX_ADDR_COUNT):
+        payload = addresses.encodeVarint(
+            len(peerList[i:i + MAX_ADDR_COUNT]))
+        for address in peerList[i:i + MAX_ADDR_COUNT]:
```

Comment:
`for stream, peer, timestamp in peerList[...`?

Reply:
you're right.